_config.yml

```diff
@@ -9,4 +9,9 @@ twitter_username: moreprideemojis
 # Build settings
 theme: minima
 #plugins:
-#  - jekyll-feed
\ No newline at end of file
+#  - jekyll-feed
+
+header_pages:
+  - README.md
+  - what-is-unicode.md
+  - campaigns.md
```

Comment:
This fixes the header navigation links, at the expense of needing to be manually updated.